### DISCUSSION TITLE
Issues with product attributes negative values

### DISCUSSION THREAD

**Discussion by Anubiella** [saleor/saleor#9382] [General]
Hi everyone,
glad to be here as always!!

The issue here is that we realized suddenly that an ordinary attribute, with these available values:
![Attributes - Ale Bike Wear 22-3-2022 11-09-30](https://user-images.githubusercontent.com/31857092/159457131-988655a2-7848-45e1-a707-45b599c5e9d2.png)
(basically negative or positive ones..)

can't be updated with negative values, let me show you..

So in the product page for instance we select a negative value from the dropdown list, as follows:
 
![product page](https://user-images.githubusercontent.com/31857092/159457953-312ed032-341d-40f2-bf09-88b16f372fbd.jpg)
and, by the time it saves, it seems everything is ok.

![GUSCIO LIGHT PACK MEN JACKET - Ale Bike Wear 22-3-2022 11-19-28](https://user-images.githubusercontent.com/31857092/159458849-9787a412-7c71-4893-8d66-63c2c623986c.png)

But as soon as you reload the page and / or navigate away and come back you can see it has been saved as
the positive one, as here:

![GUSCIO LIGHT PACK MEN JACKET - Ale Bike Wear 22-3-2022 11-20-47](https://user-images.githubusercontent.com/31857092/159459256-ba1ea456-348a-4c4e-a56f-974ad2425e2a.png)

Taking a look into the query in the dev tools I could see that the payload is correct, so for example the value sent is "-1"
BUT, in the response it send back the product object and there I could find out that the value is already incorrect, "1".

Have you ever experienced this kind of issues with attribute's negative values?
Which kind of steps / workaround I can follow?

Thanks in advance,
Lorenz

(Dashboard rel: dashboard v3.0.0-b.11 core vdev)

**Comment by karolkielecki** [comment]
Hi! @Anubiella, could you please try with the most recent stable release - core [3.1](https://github.com/saleor/saleor/releases/tag/3.1.5) + [dashboard 3.1](https://github.com/saleor/saleor-dashboard/releases/tag/3.1.2)? You're using an old 3.0 beta version.
